app/src/main/java/org/zephyrsoft/trackworktime/model/TimeSum.java

```diff
@@ -100,6 +100,11 @@ public int getAsMinutes() {
 		return hours * 60 + minutes;
 	}
 
+	public void reset() {
+		hours = 0;
+		minutes = 0;
+	}
+
 	/** test the behaviour of this class */
 	public static void test() {
```

Comment:
this method should be converted to unit tests, if there is the possibility to run tests now, but it hasn't to be with this PR

Reply:
I agree, I opted to leave it as is for now, to avoid cluttering this PR too much. I actually already have an old branch, where it's half done. Might as well finish it I guess.